### PR TITLE
button 

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
buttons

### CODE REVIEW

File: components/btn.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  .btn-outline-secondary:hover{
+    background-color: var(--btn-secondary-color);
+  }
+  .btn-outline-danger{
+    border: 1px solid var(--error-color);
+  }
+  .btn-outline-danger:hover{
+    background-color: var(--error-color);
+  }
+  .fcc-btn {
```
Comment (anshulraheja) [inline components/btn.css]: try to use a better class name

File: index.html [hunk 2]
```diff
@@ -253,6 +253,100 @@ <h1>Alert</h1>
...
+        <button class="btn btn-outline-danger">Danger</button>
+        <a target="_blank" class="fcc-btn" href="#">neoGCamp</a>
+        <a class="btn btn-primary" href="#">neoGCamp</a>
+  
+
+
+        </div>
+
+      </div>
+
```
Comment (anshulraheja) [inline index.html]: remove all unnecessary spaces in this file.

**Review by anshulraheja — APPROVED** [review]
clean UI

File: components/btn.css (hunk 1 above)
Comment (j24m) [reply components/btn.css]: +1

File: index.html (hunk 2 above)
Comment (j24m) [reply index.html]: +1

Comment (SumitSaundadkar) [reply index.html]: updated anshul thank you

**Pull request merged**